affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleConfiguration.kt

```diff
@@ -63,6 +70,27 @@ class AffectedModuleConfiguration {
      */
     var includeUncommitted: Boolean = true
 
+    /**
+     * If you want to add a custom task for impact analysis you must set the list of enum's fields
+     *
+     * Example:
+     * `build.gradle
+     *
+     *  affectedModuleDetector {
+     *       baseDir = "${project.rootDir}"
+     *       pathsAffectingAllModules = ["buildSrc/"]
+     *       specifiedBranch = "dev"
+     *       customTasks = [MyCustomTask.DETEKT_TASK] // <- list of enum fields
+     *       compareFrom = "SpecifiedBranchCommit"
+     *       includeUncommitted = false
+     *  }
+     * `
+     *
+     * @see AffectedModuleTaskType
+     * @see AffectedModuleDetectorPlugin
+     */
+    var customTasks = emptySet<AffectedModuleTaskType>()
```

Comment:
You do not have any tests for this. Please add some in https://github.com/dropbox/AffectedModuleDetector/blob/main/affectedmoduledetector/src/test/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleConfigurationTest.kt

Reply:
Yep, I added

Reply:
@joshafeinberg hey! I'll be happy to see your review again!